### PR TITLE
blog: June newsletter

### PR DESCRIPTION
**Pull request by GregSutcliffe**
I've left this really late (curses, video recording is time consuming!) ... quick review anyone? :)

### CODE REVIEW

File: _posts/2016-06-20-foreman-community-newsletter-june-2016.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+- newsletter
+excerpt: |
+  Foreman 1.12 RC1 (with Puppet 4 support!), Katello 3.0, more plugins, Red Hat
+  Summit, birthday celebrations... so much to catch up on!
+---
+
+Welcome back to the Foreman Newsletter - we hope you've had a good month! It's
+certainly been busy in the Foreman community, so let's get started...
+
+### 1.12 RC1 released
```
Comment (roidelapluie) [inline _posts/2016-06-20-foreman-community-newsletter-june-2016.md]: RC2 is out

File: _posts/2016-06-20-foreman-community-newsletter-june-2016.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+had! Watch [Twitter](https://twitter.com/ForemanProject) or the
+[events page](/events) to know whats confirmed!
+
+If you want to be involved in organising this effort, please contact
+[me](mailto:community@emeraldreverie.org) and I'll work something out with you
+- thanks!
+
+### Katello 3.0 (Saison) and 3.0.1 bugfix release
+
+The latest version of the Katello plugin hit it's milestone 3.0 release this
```
Comment (roidelapluie) [inline _posts/2016-06-20-foreman-community-newsletter-june-2016.md]: s/it's/its

File: _posts/2016-06-20-foreman-community-newsletter-june-2016.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+appropriate backups before upgrading. Do get in touch on IRC or the mailing
+lists if you have any questions!
+
+[k30]:  https://groups.google.com/d/topic/foreman-users/KlmzfqIcrRQ/discussion
+[k301]: https://groups.google.com/d/topic/foreman-users/vcjo8CW6p9I/discussion
+
+### SSL comes to (yum.|deb.|)theforeman.org
+
+Thanks to the hard work of the infra team, our website and packaging repos now
+have LetsEncrypt SSl certificates in place. This has the main benefit of
```
Comment (roidelapluie) [inline _posts/2016-06-20-foreman-community-newsletter-june-2016.md]: s/SSl/SSL/

File: _posts/2016-06-20-foreman-community-newsletter-june-2016.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+#### [Community Demo](https://youtu.be/Z2xKULg5MCA) on 9th June 2016
+
+On this episode of the Foreman Community Demo, we had a look at upcoming tools
+for flexible data exports, a new admin plugin for Hammer, some email styling
+help, new collections for Settings, and a new lock-in feature for Discovery.
+Oh, and 1.12.0 RC1 :)
+
+#### [Foreman Deep Dives: SQL Debugging for Beginners](https://www.youtu.be/kbslq-4EuyY)
+
+Chris Duryee (@beav) joined us for this Deep Dive, in which he gave us an
```
Comment (roidelapluie) [inline _posts/2016-06-20-foreman-community-newsletter-june-2016.md]: link to @beav twitter handle?

File: _posts/2016-06-20-foreman-community-newsletter-june-2016.md [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+in-depth look into SQL debugging. If you've ever wanted to know why certain
+pages are slow to load, this is one to watch!
+
+#### [Case Study on Discovery and Custom UIs with MyHeritage](https://youtu.be/p4-i4y4co0E)
+
+For this case study, we talked with Matt Nicholson (@sjoeboo, author of the
```
Comment (roidelapluie) [inline _posts/2016-06-20-foreman-community-newsletter-june-2016.md]: link to jsoeboo twitter handle?

File: _posts/2016-06-20-foreman-community-newsletter-june-2016.md [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+smart-proxy-{dhcp,dns}-infoblox plugins) about his use of Foreman at Harvard
+University Medical School, how he went about implementing the Infoblox plugins,
```
Comment (roidelapluie) [inline _posts/2016-06-20-foreman-community-newsletter-june-2016.md]: Is it a typo? Infoblox

File: _posts/2016-06-20-foreman-community-newsletter-june-2016.md [hunk 7]
```diff
@@ -0,0 +1,136 @@
...
+
+#### [Foreman San Francisco Meetup](https://plus.google.com/b/102496134326414788199/events/cngcahgf6f460tnchip0bjb9rh4)
+
+Local meetup for folks in the San Francisco area, along with those visiting for
+DevNation or Red Hat Summit. Come along and have a chat with other Foreman
+users and some of the devs!
+
+#### [Community Demo](https://plus.google.com/events/cupjll3uo0h26aqk89pa4bga5p0) on 7th July 2016
+
+The next Foreman Community Demo will be on Thursday. As ever, you can watch
```
Comment (roidelapluie) [inline _posts/2016-06-20-foreman-community-newsletter-june-2016.md]: «will be on Thursday» maybe Thursday is nos enough? e.g the first Thursday of July?

File: _posts/2016-06-20-foreman-community-newsletter-june-2016.md [hunk 8]
```diff
@@ -0,0 +1,136 @@
...
+live and partcipate via the Q&A app or via IRC, or you can catch up later on
```
Comment (roidelapluie) [inline _posts/2016-06-20-foreman-community-newsletter-june-2016.md]: missing a i in participate

File: _posts/2016-06-20-foreman-community-newsletter-june-2016.md [hunk 9]
```diff
@@ -0,0 +1,136 @@
...
+YouTube and ask questions in the mailing list thread.
+
+_(All Foreman events are now available via the [Foreman Calendar](/events))_
+
+### New plugins
+- [foreman_azure](https://github.com/theforeman/foreman_azure) - New plugin adding Azure as a Compute Resource - the [release thread](https://groups.google.com/d/topic/foreman-users/M4Yn-so6lhs/discussion) for details.
```
Comment (roidelapluie) [inline _posts/2016-06-20-foreman-community-newsletter-june-2016.md]: s/- the/- see the/

**Comment by roidelapluie** [comment]
Not a word about the forthcoming 1.11.3 and its fixed CVE ?

File: _posts/2016-06-20-foreman-community-newsletter-june-2016.md [hunk 10]
```diff
@@ -0,0 +1,136 @@
+---
+layout: post
+title: Foreman Community Newsletter - June 2016
+date: 2016-06-20
```
Comment (roidelapluie) [inline _posts/2016-06-20-foreman-community-newsletter-june-2016.md]: We are already 21st

File: _posts/2016-06-20-foreman-community-newsletter-june-2016.md (hunk 4 above)
Comment (beav) [inline _posts/2016-06-20-foreman-community-newsletter-june-2016.md]: I don't really use twitter, I would just leave it sans handle. The best way to reach me is foreman-dev channel or via the mailing list :mailbox:

Comment (roidelapluie) [inline _posts/2016-06-20-foreman-community-newsletter-june-2016.md]: yeah link to github is fine too. Just it is strange nowadays to see @username  without link :)

File: _posts/2016-06-20-foreman-community-newsletter-june-2016.md (hunk 1 above)
Comment (GregSutcliffe) [inline _posts/2016-06-20-foreman-community-newsletter-june-2016.md]: Wasn't when I wrote this :P - updated.

File: _posts/2016-06-20-foreman-community-newsletter-june-2016.md (hunk 6 above)
Comment (GregSutcliffe) [inline _posts/2016-06-20-foreman-community-newsletter-june-2016.md]: Nope, it's a company - www.infoblox.com

I'll make it a link so it's clearer

File: _posts/2016-06-20-foreman-community-newsletter-june-2016.md (hunk 7 above)
Comment (GregSutcliffe) [inline _posts/2016-06-20-foreman-community-newsletter-june-2016.md]: Yeah, same for the SF meetup, also updated.

File: _posts/2016-06-20-foreman-community-newsletter-june-2016.md (hunk 4 above)
Comment (GregSutcliffe) [inline _posts/2016-06-20-foreman-community-newsletter-june-2016.md]: It's just for IRC, not twitter - made that clear.

**Comment by GregSutcliffe** [comment]
@roidelapluie thanks! Updated.

File: _posts/2016-06-21-foreman-community-newsletter-june-2016.md [hunk 11]
```diff
@@ -0,0 +1,150 @@
...
+So far, I have events planned (but not 100% confirmed) for East-cost US, UK,
+Germany, and Israel. There'll be meetups, install days, talks and swag to be
+had! Watch [Twitter](https://twitter.com/ForemanProject) or the
+[events page](/events) to know whats confirmed!
+
+If you want to be involved in organising this effort, please contact
+[me](mailto:community@emeraldreverie.org) and I'll work something out with you
+- thanks!
+
+### Katello 3.0 (Saison) and 3.0.1 bugfix release
```
Comment (roidelapluie) [inline _posts/2016-06-21-foreman-community-newsletter-june-2016.md]: You have written:

1.11.3 bugfix released

and

1.12 RC1 and RC2 released

So maybe you can use

Katello 3.0 (Saison) and 3.0.1 bugfix released

**Comment by roidelapluie** [comment]
:+1: from me -- just posted a small optional comment

**Comment by GregSutcliffe** [comment]
thanks @roidelapluie - I'll push it in an hour or two if there's no further comments (unless someone else merges it first :P)

File: _posts/2016-06-21-foreman-community-newsletter-june-2016.md [hunk 12]
```diff
@@ -0,0 +1,150 @@
...
+other fixes. See [the announcement][mlist2] for more details.
+
+[mlist2]: https://groups.google.com/d/topic/foreman-users/3tR68n5DShw/discussion
+
+### Foreman's 7th Birthday - celebrations planned!
+
+On July 13th, Foreman will be 7 years old! Since this coincides rather nicely
+with the 1.12 release (and the Puppet 4 support) I'm currently organising a
+series of events around our various geographical centres to mark the occasion.
+So far, I have events planned (but not 100% confirmed) for East-cost US, UK,
```
Comment (mmoll) [inline _posts/2016-06-21-foreman-community-newsletter-june-2016.md]: east coast? ;)

File: _posts/2016-06-21-foreman-community-newsletter-june-2016.md [hunk 13]
```diff
@@ -0,0 +1,150 @@
...
+
+As the announcement states, this is a huge change to Katello, and upgrades
+require some planning - users are encouraged to read through the steps and take
+appropriate backups before upgrading. Do get in touch on IRC or the mailing
+lists if you have any questions!
+
+[k30]:  https://groups.google.com/d/topic/foreman-users/KlmzfqIcrRQ/discussion
+[k301]: https://groups.google.com/d/topic/foreman-users/vcjo8CW6p9I/discussion
+
+### SSL comes to (yum.|deb.|)theforeman.org
```
Comment (mmoll) [inline _posts/2016-06-21-foreman-community-newsletter-june-2016.md]: `(yum|deb).theforeman.org`, I presume?

Comment (mmoll) [inline _posts/2016-06-21-foreman-community-newsletter-june-2016.md]: oh, the last would be theforeman.org alone... looks a bit strange...

File: _posts/2016-06-21-foreman-community-newsletter-june-2016.md [hunk 14]
```diff
@@ -0,0 +1,150 @@
...
+#### [Foreman Deep Dives: SQL Debugging for Beginners](https://www.youtu.be/kbslq-4EuyY)
+
+Chris Duryee (beav on IRC) joined us for this Deep Dive, in which he gave us an
+in-depth look into SQL debugging. If you've ever wanted to know why certain
+pages are slow to load, this is one to watch!
+
+#### [Case Study on Discovery and Custom UIs with MyHeritage](https://youtu.be/p4-i4y4co0E)
+
+For this case study, we talked with Matt Nicholson
+([@sjoeboo](https://twitter.com/sjoeboo)), author of the
```
Comment (mmoll) [inline _posts/2016-06-21-foreman-community-newsletter-june-2016.md:108]: Headline doesn't match content

File: _posts/2016-06-21-foreman-community-newsletter-june-2016.md (hunk 13 above)
Comment (GregSutcliffe) [inline _posts/2016-06-21-foreman-community-newsletter-june-2016.md]: Yeah, I'm probably being too clever there :)

File: _posts/2016-06-21-foreman-community-newsletter-june-2016.md (hunk 12 above)
Comment (GregSutcliffe) [inline _posts/2016-06-21-foreman-community-newsletter-june-2016.md]: Bah, my typing is legendarily bad, right? :)

File: _posts/2016-06-21-foreman-community-newsletter-june-2016.md (hunk 14 above)
Comment (GregSutcliffe) [inline _posts/2016-06-21-foreman-community-newsletter-june-2016.md:108]: Cut'n'paste fail

**Comment by GregSutcliffe** [comment]
thanks @mmoll - this is why I don't push my own stuff... ;)

**Comment by mmoll** [comment]
merged, thanks @GregSutcliffe!

**Pull request merged**